ui/src/app/app.tsx

```diff
@@ -173,19 +173,19 @@ export class App extends React.Component<{}, {version?: Version; popupProps: Pop
     }
 
     private get archivedWorkflowsUrl() {
-        return archivedWorkflowsUrl + '/' + this.state.namespace || '';
+        return archivedWorkflowsUrl + '/' + (this.state.namespace || '');
```

Comment:
this would never have worked

* `"a" +"/" null ||  "" == "a/null"`
* `"a" + "/"(null ||  "") == "a/"`